Doppler.Sap/Services/BillingService.cs

```diff
@@ -109,6 +109,30 @@ public async Task CreateBillingRequest(List<BillingRequest> billingRequests)
             }
         }
 
+        public async Task UpdatePaymentStatus(UpdatePaymentStatusRequest updateBillingRequest)
+        {
+            try
+            {
+                var sapSystem = SapSystemHelper.GetSapSystemByBillingSystem(updateBillingRequest.BillingSystemId);
+                var validator = GetValidator(sapSystem);
+                validator.ValidateUpdateRequest(updateBillingRequest);
+                var billingRequest = GetMapper(sapSystem).MapDopplerUpdateBillingRequestToSapSaleOrder(updateBillingRequest);
+
+                _queuingService.AddToTaskQueue(
+                    new SapTask
+                    {
+                        BillingRequest = billingRequest,
+                        TaskType = SapTaskEnum.UpdateBilling
+                    }
+                );
+            }
+            catch (Exception e)
+            {
+                _logger.LogError($"Failed at update billing request for invoice: {updateBillingRequest.InvoiceId}. Error: {e.Message}");
```

Comment:
Por favor, siempre logueá la excepción entera:

```suggestion
                _logger.LogError($"Failed at update billing request for invoice: {updateBillingRequest.InvoiceId}.", e);
```